README.md

```diff
@@ -11,27 +11,20 @@ The package can be installed by adding `ex_keccak` to your list of dependencies
 ```elixir
 def deps do
   [
-    {:ex_keccak, "~> 0.1.3"}
+    {:ex_keccak, "~> 0.2.0"}
```

Comment:
I think we should bump the [major version](https://semver.org/#summary) since we're breaking compatibility here by removing the ! function. Wdyt ??

Reply:
I bumped it to 1.0.0. But I think bumping to 0.2.0 was also ok

Reply:
Actually, I did a quick search on all the libraries that are public and depend on this one, none of them use the `hash256!`. I guess there could be some private apps using that, I wouldn't mind keeping it at `0.2.0` in this case. Maybe @tsutsu can chime in here since he added the throwing function ?

Reply:
I'm using it in a private app, but I don't mind modifying the call-site. I don't have a use-case for the non-throwing variant, so this change makes sense to me.

Reply:
Awesome, then let's keep this at `0.2.0`. Sorry for the back and forth @ayrat555

Reply:
no problem. done